CMakeLists.txt

```diff
@@ -168,6 +168,9 @@ set(HERMES_ENABLE_CODE_COVERAGE OFF CACHE BOOL
 set(HERMES_ENABLE_LIBFUZZER OFF CACHE BOOL
   "Enable libfuzzer")
 
+set(HERMES_ENABLE_LPMFUZZERS OFF CACHE BOOL
+        "Enable LPM fuzzers based on libfuzzer")
```

Comment:
In this help message can you use the unabbreviated "LPM" name so people can know what it means?
Also, if you use `find_package` like I suggest below you shouldn't need this new variable.